KMaina/Fast-Food-App
# #160298106 Page to Allow an Admin Complete or Decline Orders

#### What does this PR do?
Allow an admin to complete or decline an order.

#### Description of Task to be completed?
The admin should be able to

- Approve an accepted order made 

- Decline an order made 

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-order-complete-160298106`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-order-complete-160298106`

- In the folder where you cloned the repo, you should open the UI folder.

- Open the admin-order-complete.html to see it in your browser for review.

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160298106
